test/unit/data-type.js

```diff
@@ -1079,10 +1079,6 @@ describe('SmallDateTime', function() {
       assert.throws(() => {
         TYPES.SmallDateTime.validate(new Date('Jan 1, 2080'));
       }, TypeError, 'Out of range.');
-
-      assert.throws(() => {
-        TYPES.SmallDateTime.validate(new Date('June 1, 2079'));
-      }, TypeError, 'Out of range.');
```

Comment:
Sorry, Maybe I was being confusing on this. I mean that  the test case that test against May 7, 2079 can be removed since it this date is a valid smalldatetime value.  Your original modification that change June 1, 2079 to June 7, 2079 is still a valid modification we should keep that. 
Basically this test will looks like this: 
`describe('.validate', function() {
    it.only('returns a TypeError for dates that are out of range', function() {
      assert.throws(() => {
        TYPES.SmallDateTime.validate(new Date('Dec 31, 1889'));
      }, TypeError, 'Out of range.');

      assert.throws(() => {
        TYPES.SmallDateTime.validate(new Date('Jan 1, 2080'));
      }, TypeError, 'Out of range.');

      assert.throws(() => {
        TYPES.SmallDateTime.validate(new Date('June 7, 2079'));
      }, TypeError, 'Out of range.');
    });
  });`